CONTRIBUTING.md

```diff
@@ -30,9 +38,50 @@ The following are required to build the devfile index container image containing
     - This will also validate the devfiles in this repository, making sure they conform to a minimum standard.
     - This step will also be run in the PR build for the repository.
 
-5) (Optional) Push the container image to a container registry and [deploy](https://github.com/devfile/registry-support#deploy) a custom devfile registry on to Kubernetes or OpenShift. Test odo with the devfile stack hosted in that registry.
-  
-    - For now, strictly an optional requirement, but recommended.
-    - PR tests will also verify this.
+5) Open a pull request against this repository with a brief description of the change.
+
+
+### Updating
+
+Updating an existing devfile stack is relatively straightforward:
+
+1) Find the stack under the `stacks/` folder that you wish to update
+2) Make the necessary changes to the stack (such as updating image tags, commands, starter projects, etc)
```

Comment:
Not a big deal, but parentheses do imply a lack of importance. Consider rephrasing to:

`Make the necessary changes to the stacks, such as: updating image tags, commands, starter projects, etc.`

Reply:
Huh, TIL

Reply:
Fixed